### PR TITLE
feat: set real time series data

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
스크린샷 2023-06-12 오후 8 49 11

### CODE REVIEW

**Review by sulmoJ — APPROVED** [review]
LGTM!

File: apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue [hunk 1]
```diff
@@ -133,126 +139,98 @@ interface InnerChartData {
...
-        severity: status.label,
-        value: state.severityData.find((data) => data.severity === status.name)?.fail_finding_count ?? 0,
+        value: state.checkCount?.[status.name],
         pieSettings: {
             fill: color(status.color),
             stroke: color(status.color),
         },
     }))),
+    innerChartData: computed<InnerChartData[]>(() => {
+        const targetDataList = state.data?.filter((d) => d.date === state.dateRange.end && d.key === 'fail_finding_count') ?? [];
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue:147]: description: Only the last month's data is used for the inner chart

File: apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue [hunk 2]
```diff
@@ -133,126 +139,98 @@ interface InnerChartData {
...
+        if (!state.data) return undefined;
+        const targetDataList = state.data.filter((d) => d.date === state.dateRange.start && d.key === 'compliance_count');
+        return sum(targetDataList.map((d) => d.value));
+    }),
+    complianceCount: computed<number|undefined>(() => {
+        if (!state.data) return undefined;
+        const targetDataList = state.data.filter((d) => d.date === state.dateRange.end && d.key === 'compliance_count');
+        return sum(targetDataList.map((d) => d.value));
+    }),
+    complianceCountHelpText: computed<TranslateResult|undefined>(() => {
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue:169]: suggestion: Please change the state name more meaningful~ such as complianceCountComparingMessage

File: apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue [hunk 3]
```diff
@@ -133,126 +123,98 @@ interface InnerChartData {
...
+            || state.complianceCount === state.prevComplianceCount
+        ) return undefined;
+        if (state.prevComplianceCount < state.complianceCount) {
+            return i18n.t('DASHBOARDS.WIDGET.COMPLIANCE_CHECK_STATUS.MORE_THAN_PREV_MONTH');
+        }
+        return i18n.t('DASHBOARDS.WIDGET.COMPLIANCE_CHECK_STATUS.LESS_THAN_PREV_MONTH');
+    }),
+    checkCount: computed(() => {
+        if (!state.data) return undefined;
+        const targetDataList = state.data.filter((d) => d.date === state.dateRange.end);
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/asset-widgets/compliance-check-status/ComplianceCheckStatusWidget.vue:181]: description: Only the last month's data is used for the check count

**Review by WANZARGEN — APPROVED** [review]
Wow.... I think we need more understand about the spec of this widget!! :)

**Pull request merged**